packages/vue/src/components/molecules/SfModal/SfModal.js

```diff
@@ -0,0 +1,78 @@
+export default {
+  name: "SfModal",
+  props: {
+    /**
+     * Cross closing modal button
+     */
+    cross: {
+      type: Boolean,
+      default: true
+    },
+    /**
+     * Visibility of the modal
+     */
+    visible: {
```

Comment:
this should be handled by outside v-if/v-show. No need to have internal prop for this